src/Hubbup.MikLabelModel/Hubbup.MikLabelModel.csproj

```diff
@@ -1,13 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <Import Project="..\..\eng\Versions.props" />
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
why are we downgrading it ?

Reply:
Because it was not compatible with the ML package used in the other project. I will later need to upgrade the ML package versions alongside the target framework for this.

Reply:
I'll spend another cycle on upgrading the target framework later